conf.d/main

```diff
@@ -9,7 +9,12 @@ apt install --assume-yes wireguard
 # webserver related
 mkdir -p /var/www/wireguard/logs
 chown www-data:www-data /var/www/wireguard/logs
-lighty-disable-mod ssl
+
+service lighttpd restart
+lighty-enable-mod wireguard
+service lighttpd force-reload
```

Comment:
Same with these 3 lines.

As it doesn't actually use the webserver during build time (like some other appliances do; e.g. for curl install of a PHP app). As such, strictly speaking, you don't even need to start lighty. Having said that, perhaps starting then stopping it is a good idea to test that the config is correct?! Although if that was why you're doing this, you could just use the command to test lighty config (in the conf script). I.e. `lighttpd -tt -f /etc/lighttpd/lighttpd.conf`

Reply:
The force reload did come up on screen in the logs while building the appliance, I don't believe it stopped the build but it did say to run `service lighttpd force reload` after running the `lighty-enable-mod wireguard` . After the build, I also re ran both `lighty-enable-mod wireguard` and `lighty-enable-mod ssl` but the terminal replied they were already enabled.

Reply:
I went through and added the test script per PR review comment. I also went through and removed the lines restarting/reloading the lighttpd service in conf.d/main and retested appliance.